Dockerfile

```diff
@@ -0,0 +1,18 @@
+ARG PYTHON_VERSION=3.11-slim
+
+FROM python:${PYTHON_VERSION} AS base
+ENV PYTHONUNBUFFERED=1
+WORKDIR /app
+COPY requirements.txt /app/
+RUN pip install --no-cache-dir -r requirements.txt
+
+FROM base AS build
+COPY . /app
+RUN python manage.py migrate
```

Comment:
Running `python manage.py migrate` in the build stage might not be ideal if the database is not available during the build process. Consider running migrations as part of the entrypoint or command in the final stage.

Reply:
Fixed